examples/platform/silabs/SiWx917/SiWx917/rsi_if.c

```diff
@@ -57,6 +57,12 @@ StaticTask_t driverRsiTaskBuffer;
 /* Declare a variable to hold the data associated with the created event group. */
 StaticEventGroup_t rsiDriverEventGroup;
 
+/* Declare a flag to differentiate between after boot-up first IP connection or reconnection */
+bool is_wifi_disconnection_event = false;
```

Comment:
should these new variables be static?

Reply:
Yes, declared **is_wifi_disconnection_event** variable as a static.